nservicebus/lifecycle/ineedinitialization.md

```diff
@@ -0,0 +1,28 @@
+---
+title: INeedInitialization
+summary: An interface that allows you to hook into the very beginning of the bus creation sequence of NServiceBus
+tags:
+ - lifecycle
+related:
+ - samples/startup-shutdown-sequence
+---
+
+Classes that implement `NServiceBus.INeedInitialization` are created and called as one of the first steps in the bus creation lifecycle. Use `INeedInitialization` to register components that will be used later in the bus creation lifecycle. 
+
+NOTE: In V3 of NServiceBus, this interface is found in the `NServiceBus.Config` namespace. In V4 both interfaces are available but the old one is marked as obsolete. As of V5 use of the `NServiceBus.Config.INeedInitialization` interface will cause a compile-time error. In V4 (which has both interfaces), all instances of `NServiceBus.Config.INeedInitialization` are created and called and then all instances of `NServiceBus.INeedInitialization` are created and called.
```

Comment:
i know this is minor but everywhere else in the doco we use "Version X" not "VX"